src/main/resources/web/README.md

```diff
@@ -0,0 +1,73 @@
+# dwrtc.js
+
+JS helper for the [DWRTC server](https://github.com/dwrtc/dwrtc) to initiate WebRTC connections over a P2P network.
+
+## Prerequisites
+
+To be able to run dwrtc.js a running [DWRTC server](https://github.com/dwrtc/dwrtc) (check [dwrtc.net](dwrtc.net) for available servers or deploy your own) and [simple-peer](https://github.com/feross/simple-peer) are required.
+
+## Usage
+
+```html
+<html>
+  <body>
+    <div class="container"></div>
+
+    <script src="simple-peer.js"></script>
+    <script src="dwrtc.js"></script>
+    <script>
+        const dwrtc = new DWRTC(initiator, initialPeerId, webSocketUrl)
```

Comment:
I would explain the parameters, e.g. (still needs formatting)

* `initiator`: true if this peer is the calling peer
* `initialPeerId`: partner session ID if this is the calling peer. Otherwise undefined
* `webSocketUrl`: URL to the WebSocket endpoint